### ISSUE TITLE
Venn Diagram Interpretation

### ISSUE THREAD

**Issue by charleshalldesign** [w3c/coga#129]
**Issue**

The Venn Diagram appearing at the end of [§2.2 Background about People with Learning and Cognitive Disabilities and the Web](https://www.w3.org/TR/coga-usable/#background) seems problematic.

**Opinion & Interpretation**

At their cores, accessibility is the absence of barriers, and usability is the quality of use. If it is not accessible first, it cannot be used, and therefore the quality cannot be measured. So accessibility should be a much larger circle and encompass usability. 

Also, the diagram is interpreted by the relative scale of each area. So, the fact that the overlapping area of usable & accessible is so small, perpetuates the myth that only a small number of things can achieve both.

**Suggestion**

The text content of this section sufficiently describes the background. So the diagram could be removed with no impact on the content.

**Comment by rachaelbradley** [comment]
Thank you for this great suggestion. We have removed the Venn diagram.
